### PR TITLE
Fix/20800 style issues budget page

### PR DESCRIPTION
**Pull request by HDinger**
Fixed design issues on the budget page, such as aligning fields and labels.

https://community.openproject.org/work_packages/20800

### CODE REVIEW

**Comment by floriank** [comment]
Are you sure you want the locales in there? :wink:

**Comment by HDinger** [comment]
@NobodysNightmare Could you please have a look at this PR?

File: app/assets/stylesheets/costs/costs_legacy.css [hunk 1]
```diff
@@ -97,6 +102,24 @@ td.comment input {
   cursor: pointer;
 }
 
+.budget_fieldset{
+  margin: 8px 2px;
+}
+
+.budget_table_inputFields{
```
Comment (NobodysNightmare) [inline app/assets/stylesheets/costs/costs_legacy.css]: I'd rather stick to a uniform casing strategy
- `snake_case`
- `camelCase`
- `dash-case`

`but_notMixed`.

Thinking about our other css classes it should probably be `budget-table--input-field`, shouldn't it?

File: app/views/cost_objects/_labor_budget_item.html.erb [hunk 2]
```diff
@@ -57,7 +57,7 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
     <%= observe_field( "#{id_prefix}_user_id", :url => {:action => :update_labor_budget_item, :project_id => @project.id}, :with => "'user_id=' + encodeURIComponent(value) + '&hours=' + encodeURIComponent(document.getElementById('#{id_prefix}_hours').value) + '&fixed_date=' + encodeURIComponent(document.getElementById('cost_object_fixed_date').value) + '&element_id=#{id_prefix}'") %>
     <%= observe_field( "#{id_prefix}_hours", :frequency => 1, :url => {:action => :update_labor_budget_item, :project_id => @project.id}, :with => "'user_id=' + encodeURIComponent(document.getElementById('#{id_prefix}_user_id').value) + '&hours=' + encodeURIComponent(value) + '&fixed_date=' + encodeURIComponent(document.getElementById('cost_object_fixed_date').value) + '&element_id=#{id_prefix}'") %>
   </td>
-  <td class="delete">
+  <td class="delete budget_table_fields">
```
Comment (NobodysNightmare) [inline app/views/cost_objects/_labor_budget_item.html.erb]: just noticed that you added a class to every `td` here.
Wouldn't it be better to have a CSS class on the table and derive rules from that?
But maybe I am overlooking something...

**Comment by NobodysNightmare** [comment]
There are still some issues for me left in Chrome:
problems
- **red**: the baselines of the textboxes/selects do not match the baselines of the plain text labels
- **pink**: For some selects the lower half of the text is cut (I think a line-height issue)
- **green-ish**: The margin above the plus icon is very small/does not exist

**Comment by NobodysNightmare** [comment]
The submit button does not have a highlight color (class `-highlight`):

image

**Pull request merged**